DESCRIPTION

```diff
@@ -17,7 +17,7 @@ Imports:
     devtools,
     httr
 Remotes:
-    annecori/EpiEstim,
+    annecori/EpiEstim@new-version,
     nickreich/coarseDataTools@hackout3,
```

Comment:
This branch no longer exists as the code has been merged into the master branch (https://github.com/nickreich/coarseDataTools/pull/50) (and hopefully will be on CRAN soon)

Reply:
Good point. Let's get `new-version` of `EpiEstim` merged and then hopefully we can remove both of these specific branches at the same time, and then merge this PR.

I feel that keeping EpiEstimApp up to date with EpiEstim during development will always be a pain - they potentially both need simultaneous changes, e.g. if a function of EpiEstim changes. Not sure how best to manage this other than having temporary changes like this...